OptimizelySDKCore/OptimizelySDKCore/OPTLYBaseCondition.m

```diff
@@ -132,7 +140,7 @@ - (nullable NSNumber *)evaluateMatchTypeGreaterThan:(NSDictionary<NSString *, NS
     // check if user attributes contain a value greater than our value
     
     // check if condition value is invalid
-    if (self.value == nil || [self.value isKindOfClass:[NSNull class]] || ![self.value isValidNumericAttributeValue]) {
+    if (![self.value isValidGTLTMatchTypeValue]) {
```

Comment:
Ah, nice catch!

Reply:
Before we return null, can we log a message
```
@"[AUDIENCE EVALUATOR] Audience condition %@ has an unsupported condition value. You may need to upgrade to a newer release of the Optimizely SDK."
```
at warning level?

(Same thing when checking `self.value` in `evaluateMatchTypeExact`, `evaluateMatchTypeSubstring`, `evaluateMatchTypeLessThan`, and `evaluateCustomMatchType`.)

Reply:
Recommended changes made, please review.

Reply:
Thanks!